src/main/java/nl/ordina/jobcrawler/service/ScraperService.java

```diff
@@ -46,7 +47,8 @@ public ScraperService(VacancyService vacancyService, LocationService locationSer
         this.jobBirdScraper = jobBirdScraper;
     }
 
-    @Scheduled(cron = "0 0 12,18 * * *")
+    @PostConstruct
```

Comment:
Please remove this PostConstruct annotation and only enable the Scheduled annotation :-)